### ISSUE TITLE
alg should not be mandatory in cose headers

### ISSUE THREAD

**Issue by OR13** [cose-wg/HPKE#61]
https://mailarchive.ietf.org/arch/msg/cose/GrvF6Dy8NkFbB79nHdzOdo0cGfc/

**Comment by hannestschofenig** [comment]
I agree that the alg parameter cannot be mandatory. With all the discussions we had so far I guess we all realize by now that even using the alg parameter is a bad idea

**Comment by hannestschofenig** [comment]
In the latest version of the draft the alg parameter is not mandatory anymore.
